src/components/views/rooms/RoomList.tsx

```diff
@@ -117,9 +118,24 @@ const TAG_AESTHETICS: ITagAestheticsMap = {
         sectionLabel: _td("People"),
         isInvite: false,
         defaultHidden: false,
-        addRoomLabel: _td("Start chat"),
+        addRoomLabel: _td("Add people"),
         onAddRoom: (dispatcher?: Dispatcher<ActionPayload>) => {
-            (dispatcher || defaultDispatcher).dispatch({action: 'view_create_chat'});
+            const activeSpace = SpaceStore.instance.activeSpace;
+            const dis = dispatcher || defaultDispatcher;
+
+            if (activeSpace && !activeSpace.canInvite(MatrixClientPeg.get().getUserId())) {
+                Modal.createDialog(ErrorDialog, {
+                    title: _t("You cannot invite people to this space"),
+                    description: _t("You do not have sufficient permissions to invite people to this space"),
+                });
```

Comment:
perhaps we should show the Modal here

![image](https://user-images.githubusercontent.com/2403652/123654420-f6be7a80-d825-11eb-8410-f17fef10a5de.png)
![image](https://user-images.githubusercontent.com/2403652/123654454-0211a600-d826-11eb-8c5e-9692a9891e57.png)